app/src/main/res/values/strings.xml

```diff
@@ -213,6 +213,7 @@
   <string name="login_to_your_account">Login to your account</string>
   <string name="send_log_file">Send log file</string>
   <string name="send_log_file_description">Send log file to developers via email</string>
+  <string name="no_web_browser">No app found to open URL</string>
```

Comment:
Could we perhaps get a more detailed explanation, like "No web browser found to open URL"?